src/containers/subjects/SubjectWGSTNLaunch/index.tsx

```diff
@@ -0,0 +1,311 @@
+import React, { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+import { InputText } from 'primereact/inputtext';
+import { DataTable } from 'primereact/datatable';
+import { Column } from 'primereact/column';
+import { Button } from 'primereact/button';
+import { Dialog } from 'primereact/dialog';
+import { RadioButton } from 'primereact/radiobutton';
+
+import CircularLoaderWithText from '../../../components/CircularLoaderWithText';
+import StyledJsonPretty from '../../../components/StyledJsonPretty';
+
+import './index.css';
+import { usePortalMetadataAPI } from '../../../api/metadata';
+import { invokeWGSTNWorkflow } from './aws';
+import { FastqRow, FASTQPairingPayload, usePortalSubjectParingAPI } from '../../../api/pairing';
+
+const metadataHeaderToDisplay: string[] = [
+  'subject_id',
+  'sample_id',
+  'library_id',
+  'external_subject_id',
+  'external_sample_id',
+  'type',
+  'phenotype',
+  'project_name',
+];
+
+type Props = { subjectId: string };
+export default function SubjectWGSTNLaunch({ subjectId }: Props) {
+  const [isConfirmDialogOpen, setIsConfirmDialogOpen] = useState<boolean>(false);
+  const [isLaunch, setIsLaunch] = useState<boolean>(false);
+
+  const [input, setInput] = useState<FASTQPairingPayload | null>(null);
+
+  const workflowTriggerRes = useQuery(
+    ['wgs-tn-invoke', input],
+    async () => {
+      if (input) await invokeWGSTNWorkflow(input);
+    },
+    {
+      enabled: isLaunch && !!input,
+    }
+  );
+
+  const pairingOptions = usePortalSubjectParingAPI({
+    apiConfig: {
+      header: { 'Content-Type': 'application/json' },
+      body: [subjectId],
+    },
+  });
+
+  const metadataUseQueryRes = usePortalMetadataAPI({
+    additionalPath: '?workflow=clinical&workflow=research&phenotype=tumor&phenotype=normal',
```

Comment:
Ah. `additionalPath` is used here. Perhaps make it optional in `usePortalMetadataAPI(...)` I reckon. I leave it to you!